docs/docusaurus/docs/core/define_expectations/_retrieve_a_batch_of_test_data/_from_a_batch_definition.md

```diff
@@ -0,0 +1,100 @@
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+import PrereqPython from '../../_core_components/prerequisites/_python_installation.md';
+import PrereqGxInstallation from '../../_core_components/prerequisites/_gx_installation.md';
+import PrereqDataContext from '../../_core_components/prerequisites/_preconfigured_data_context.md';
+import PrereqDataSourceAndAssetConnectedToData from '../../_core_components/prerequisites/_data_source_asset_and_batch_definition.md';
+
+Batch Definitions both organize a Data Asset's records into Batches and provide a method for retrieving those records.  Any Batch Definition can be used to retrieve a Batch of records for use in testing Expectations or data exploration.
+
+## Prerequisites
+
+- <PrereqPython/>.
+- <PrereqGxInstallation/>.
+- <PrereqDataContext/>.  These examples assume the variable `context` contains your Data Context.
+- <PrereqDataSourceAndAssetConnectedToData/>.
+
+<Tabs>
+
+<TabItem value="procedure" label="Procedure">
+
+1. Retrieve your Batch Definition.
+
+   Update the values of `data_source_name`, `data_asset_name`, and `batch_definition_name` in the following code and execute it to retrieve your Batch Definition from the Data Context:
+
+   ```python title="Python" name="docs/docusaurus/docs/core/define_expectations/_retrieve_a_batch_of_test_data/_from_a_batch_definition.py - retrieve Batch Definition"
+   ```
+
+2. Optional. Specify the Batch to retrieve.
+
+   Some Batch Definitions can only provide a single Batch.  Whole table batch definitions on SQL Data Assets, file path and whole directory Batch Definitions on filesystem Data Assets, and all Batch Definitions for dataframe Data Assets will provide all of the Data Asset's records as a single Batch.  For these Batch Definitions there is no need to specify which Batch to retrieve because there is only one available.
+
+   Yearly, monthly, and daily Batch Definitions subdivide the Data Asset's records by date.  This allows you to retrieve the data corresponding to a specific date from the Data Asset.  If you do not specify a Batch to retireve, these Batch Definitions will return the first valid Batch they find.  By default, this will be the most recent Batch (sort ascending) or the oldest Batch if the Batch Definition has been configured to sort descending.
```

Comment:
`retireve` 😄

Reply:
Good grief.  Thanks.  I really wish pycharm had better spellchecking (and grammarly support).  😅

Reply:
I have misspelled `checkpoint` as `checkpiont` in almost every PR I've pushed 😄